### PR TITLE
added swagger configuration

### PR DESCRIPTION
**Pull request by payaltikyaniongraph**

### CODE REVIEW

File: src/main/java/com/notification/config/TripplyBookingConfig.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        server.setDescription("Development");
+
+        Contact contact = new Contact();
+        contact.setName("Tripply");
+        contact.setEmail("tripply@ongraph.com");
+
+        Info information = new Info()
+                .title("Tripply - Hotel Booking System")
+                .version("1.0")
+                .description("This API exposes endpoints for notification service")
```
Comment (hitsa29) [inline src/main/java/com/notification/config/TripplyBookingConfig.java:26]: Is swagger configured only for specific APIs or for whole project. from description it seems like it's created only for notification service.

File: src/main/java/com/notification/config/TripplyBookingConfig.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.beans.factory.annotation.Configurable;
+
+import java.util.List;
+
+@Configurable
+public class TripplyBookingConfig {
+
+    public OpenAPI defineOpenApi(){
+        Server server = new Server();
+        server.setUrl("http://localhost:8087");
```
Comment (hitsa29) [inline src/main/java/com/notification/config/TripplyBookingConfig.java:16]: please take url and environment from environment variables.

File: src/main/java/com/notification/config/TripplyBookingConfig.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        server.setDescription("Development");
+
+        Contact contact = new Contact();
+        contact.setName("Tripply");
+        contact.setEmail("tripply@ongraph.com");
+
+        Info information = new Info()
+                .title("Tripply - Hotel Booking System")
+                .version("1.0")
```
Comment (hitsa29) [inline src/main/java/com/notification/config/TripplyBookingConfig.java:25]: version should be also there in env variables

File: src/main/java/com/notification/config/TripplyBookingConfig.java (hunk 1 above)
Comment (payaltikyaniongraph) [reply src/main/java/com/notification/config/TripplyBookingConfig.java:26]: The swagger is configured for all apis of notification service. Likewise, if you see auth and booking service, we have swagger configuration for them also.

**Review by Jatin-Ongraph — APPROVED** [review]

**Pull request merged**